tests/integration/placeholder.cpp

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Remove, we dont need placeholders.

Reply:
Only thing blocking the merge is the removal of this placeholder empty file.

Reply:
I couldn't have integration tests binary without any source file.

Reply:
But then I would say better is to postpone the creation of the integration test binary instead of having shady dummy files, right? Anyway, to not keep this endless loop going. Please avoid adding dummy files in the future!